src/styles/body.scss

```
@@ -0,0 +1,14 @@
+.main {
+  @include center;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  margin: 50px auto;
+  padding: 0 16px;
+
+  @media screen and (min-width: 1200px) {
+    & {
+      max-width: 1200px;
+      padding: 0 104px;
+    }
```

якщо media запит вкладений в main, то не потрібно прописувати `&`
```suggestion
  @media screen and (min-width: 1200px) {
      max-width: 1200px;
      padding: 0 104px;
```